lib/container_orchestrator/object_definition.rb

```diff
@@ -47,6 +47,27 @@ def deployment_definition(name)
             }
           ]
         end
+      end.tap do |deployment|
```

Comment:
it may be time to graduate to a local variable and adding stuff to it.
Or a method for each block and 3 merges in a row